File: Javascript/Miles per gallon to kilometers per liter.js
// https://www.codewars.com/kata/miles-per-gallon-to-kilometers-per-liter/train/javascript


function converter (mpg) {
    const gallon = 4.54609188;
    const kilometers = 1.609344;
    return (mpg * kilometers / gallon).toFixed(2);
  }

File: C#/Miles per gallon to kilometers per liter.cs
// https://www.codewars.com/kata/miles-per-gallon-to-kilometers-per-liter/train/csharp
using System;
using System.Collections.Generic;
public static class Kata
{
	public static double Converter(int mpg)
	{
		double gallon = 4.54609188;
    double kilometers = 1.609344;
    return Math.Round(mpg * kilometers / gallon, 2);
	}
}